### PR TITLE
Release notes v20.1.0 beta.4

### PR DESCRIPTION
**Pull request by taroface**

### CODE REVIEW

**Comment by taroface** [comment]
@jseldess Are there any docs issues or larger announcements to include in this one?

Note: It looks like the release hasn't been finalized yet, so there might be more items to add.

File: releases/v20.1.0-beta.4.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+### Bug fixes
+
+- Fixed a crash when [`IMPORT`](../v20.1/import.html)ing a table without a table definition. [#46193][#46193]
+- Queries with qualified stars that refer to tables in outer scopes now work. [#46233][#46233]
+- Fixed an incorrect "no data source matches prefix" error in some cases involving subqueries that use views. [#46226][#46226]
+- Previously, the `experimental_strftime` and `experimental_strptime` [builtin](../v20.1/functions-and-operators.html) functions used the non-POSIX standard %f for nanosecond display. However, as CockroachDB only supports up to microsecond precision and Python's `strftime` has %f to microsecond, we have similarly switched %f to use microsecond instead of nanosecond precision. [#46263][#46263]
+- Added a check that detects invalid sequence numbers in the RocksDB write-ahead log and returns an error during start instead of applying the invalid log entries. [#46328][#46328]
+- [Follower reads](../v20.1/follower-reads.html) that hit intents no longer have a chance of entering an infinite loop. This bug was present in earlier versions of the v20.1 release. [#46234][#46234]
```
Comment (rmloveland) [inline releases/v20.1.0-beta.4.md]: Suggest "fixed a bug where".

Also it makes me wonder if this bug was *only* in earlier 20.1 releases?  This wording could mean it's also present elsewhere.

File: releases/v20.1.0-beta.4.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+- Fixed an internal error that could occur when an aggregate inside the right-hand side of a `LATERAL` join was scoped at the level of the left-hand side. [#46227][#46227]
+- Fixed an error that incorrectly occurred when an aggregate was used inside the `WHERE` or `ON` clause of a subquery but was scoped at an outer level of the query. [#46227][#46227]
```
Comment (rmloveland) [inline releases/v20.1.0-beta.4.md]: Could link to our subquery docs: https://www.cockroachlabs.com/docs/v19.2/subqueries.html

File: releases/v20.1.0-beta.4.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- Queries with qualified stars that refer to tables in outer scopes now work. [#46233][#46233]
+- Fixed an incorrect "no data source matches prefix" error in some cases involving subqueries that use views. [#46226][#46226]
+- Previously, the `experimental_strftime` and `experimental_strptime` [builtin](../v20.1/functions-and-operators.html) functions used the non-POSIX standard %f for nanosecond display. However, as CockroachDB only supports up to microsecond precision and Python's `strftime` has %f to microsecond, we have similarly switched %f to use microsecond instead of nanosecond precision. [#46263][#46263]
+- Added a check that detects invalid sequence numbers in the RocksDB write-ahead log and returns an error during start instead of applying the invalid log entries. [#46328][#46328]
+- [Follower reads](../v20.1/follower-reads.html) that hit intents no longer have a chance of entering an infinite loop. This bug was present in earlier versions of the v20.1 release. [#46234][#46234]
+- Fixed an internal error that could occur when an aggregate inside the right-hand side of a `LATERAL` join was scoped at the level of the left-hand side. [#46227][#46227]
```
Comment (rmloveland) [inline releases/v20.1.0-beta.4.md]: Could link to our join docs: https://www.cockroachlabs.com/docs/v19.2/joins.html

File: releases/v20.1.0-beta.4.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+- Fixed an error that incorrectly occurred when an aggregate was used inside the `WHERE` or `ON` clause of a subquery but was scoped at an outer level of the query. [#46227][#46227]
+- Reverted performance improvements to incremental [`BACKUP`](../v20.1/backup.html)s until a potential correctness issue is addressed. [#46385][#46385]
+- [CDC](../v20.1/change-data-capture.html) no longer combines with long running transactions to trigger an assertion. Previously, this could crash a server if the right sequence of events occurred. This was typically rare, but was much more common when CDC was in use. [#46391][#46391]
```
Comment (rmloveland) [inline releases/v20.1.0-beta.4.md]: Might be worth revising to "fixed a bug where CDC ..."

File: releases/v20.1.0-beta.4.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+- Fixed a race in the [vectorized execution engine](../v20.1/vectorized-execution.html). [#46360][#46360]
+- Fixed a rare bug causing transactions that have performed schema changes to deadlock after they restart. [#46384][#46384]
+
+### Doc updates
+
+Docs team: Please add these manually.
```
Comment (rmloveland) [inline releases/v20.1.0-beta.4.md]: In the past I have filled this in by running `git log` in the docs repo and checking for stuff that merged between the previous beta and this one.

**Pull request merged**